runbotics/runbotics-orchestrator-ui/src/src-app/components/Tile/ProcessTile/ProcessTile.tags.styles.ts

```diff
@@ -0,0 +1,96 @@
+import ExpandLessIcon from '@mui/icons-material/ExpandLess';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import { Chip, Typography } from '@mui/material';
+import styled from 'styled-components';
+
+export const StyledChip = styled(Chip)`
+    margin: 0 8px 8px 0;
+`;
+
+export const StyledTypography = styled(Typography)(({ theme }) => `
+    && {
+        color: ${theme.palette.grey[500]};
+        text-transform: lowercase;
+        position: relative;
+        top: 7px;
+        user-select: none;
+
+        &:hover {
+            cursor: pointer;
+        }
+    }
+`);
+
+export const StyledMoreIcon = styled(ExpandMoreIcon)(({ theme }) => `
+    && {
+        position: relative;
+        top: 10px;
+        color: ${theme.palette.grey[500]};
+    }
+`);
+
+export const StyledLessIcon = styled(ExpandLessIcon)(({ theme }) => `
+    && {
+        position: relative;
+        top: 10px;
+        color: ${theme.palette.grey[500]};
+    }
+`);
+
+export const Container = styled.div`
+    display: flex;
+    flex-wrap: wrap;
+    position: relative;
+    top: -8px;
+    height: 20px;
```

Comment:
have you tested app in general on responsive screens? (it doesn't have to be perfect on mobile but should look great both at HD and 2K resolutions

Reply:
Yes, I tested it. It looks good for me. :)